wnt/protocol_buffers_files/commons.proto

```diff
@@ -35,6 +35,22 @@ enum ScratchpadAction {
     SCRATCHPAD_ACTION_LEGACY = 5;
 }
 
+enum StackProfile {
+    PROFILE_24 = 1;
+    PROFILE_868 = 2;
+    PROFILE_915 = 3;
+    PROFILE_870 = 4;
+    PROFILE_917 = 5;
+    PROFILE_RESERVED_1 = 6;
```

Comment:
Should we leave the `_RESERVED` ones out? They are not real stack profiles anyway.

Reply:
They have been in use at least internally at some point, so need to have them so we are backwards compatible to any version.